netbox/extras/forms/bulk_import.py

```diff
@@ -140,19 +141,28 @@ class Meta:
 
 
 class WebhookImportForm(NetBoxModelImportForm):
+
+    class Meta:
+        model = Webhook
+        fields = (
+            'name', 'payload_url', 'http_method', 'http_content_type', 'additional_headers', 'body_template',
+            'secret', 'ssl_verification', 'ca_file_path', 'tags'
+        )
+
+
+class EventRuleImportForm(NetBoxModelImportForm):
     content_types = CSVMultipleContentTypeField(
         label=_('Content types'),
         queryset=ContentType.objects.all(),
-        limit_choices_to=FeatureQuery('webhooks'),
+        limit_choices_to=FeatureQuery('eventrules'),
```

Comment:
This will need to be updated for #14153 and to reference "event rules" as a feature.